modules/launcher/src/main/java/org/mule/runtime/module/launcher/DefaultMuleContainer.java

```diff
@@ -351,8 +364,6 @@ protected void doShutdown() throws MuleException {
   }
 
   public void stop() throws MuleException {
-    MuleContainerBootstrap.dispose();
-
```

Comment:
This is now called from the [Wrapper impl](https://github.com/mulesoft/mule/pull/12618/files#diff-f6bec6cc13ec17261d65597acea20d56dbaf9efb2104c24c1d4a1df42396ce79R35) when it gets signaled to stop.